dashboard/app/models/ability.rb

```diff
@@ -61,7 +61,7 @@ def initialize(user)
     cannot :index, Level
 
     # If you can see a level, you can also do these things:
-    can [:embed_level, :get_rubric], Level do |level|
+    can [:embed_level, :get_rubric, :get_serialized_maze], Level do |level|
```

Comment:
Since the Javabuilder backend isn't a verified member of the pilot will it be able to see the level?

Reply:
(i.e. question based on the comment above)

Reply:
My understanding is this is for a generic 'see a level' permission, not a specific level. I have a unit test below that successfully accesses the route without any login, so I think it will work.